docs/community/governance.md

```diff
@@ -90,36 +90,42 @@ One of their primary responsibilities is merging pull requests after all require
 
-## Area Experts
+## Working Groups (WGs)
 
-Area Experts are people with knowledge of specific components or technologies in the PowerShell domain. They are responsible for code reviews, issue triage, and providing their expertise to others.
+[Working Groups (WGs)][wg] are collections of contributors with knowledge of specific components or
+technologies in the PowerShell domain.
+They are responsible for issue triage/acceptance, code reviews, and providing their expertise to
+others in issues, PRs, and RFC discussions.
 
 They have [write access](https://help.github.com/articles/permission-levels-for-an-organization-repository/) to the PowerShell repository which gives them the power to:
 
 1. `git push` to all branches *except* `master`.
 1. Merge pull requests to all branches *except* `master` (though this should not be common given that [`master`is the only long-living branch](../git/README.md#understand-branches)).
 1. Assign labels, milestones, and people to [issues](https://guides.github.com/features/issues/).
 
-A list of Area Experts can be found [here][experts].
-
-### Area Expert Responsibilities
+### Working Group Responsibilities
 
-If you are an Area Expert, you are expected to be actively involved in any development, design, or contributions in your area of expertise.
+If you are a member of a Working Group, you are expected to be actively involved in any development, design, or contributions in the focus area of the WG.
+More information on the responsibilities of Working Groups can be found [here][wg],
+while current WG definitions and membership can be found [here][wg-definitions].
 
-If you are an Area Expert:
+If you are a Working Group member:
 
-1. **DO** assign the [correct labels][issue-process]
-1. **DO** assign yourself to issues labeled with your area of expertise
-1. **DO** code reviews for issues where you're assigned or in your areas of expertise.
-1. **DO** reply to new issues and pull requests that are related to your area of expertise
-  (while reviewing PRs, leave your comment even if everything looks good - a simple "Looks good to me" or "LGTM" will suffice, so that we know someone has already taken a look at it).
-1. **DO** make sure contributors are following the [contributor guidelines](../../.github/CONTRIBUTING.md).
-1. **DO** ask people to resend a pull request, if it [doesn't target `master`](../../.github/CONTRIBUTING.md#lifecycle-of-a-pull-request).
+1. **DO** triage and contribute to discussions in issues and PRs that have `WG-*` labels assigned
+1. **DO** regularly communicate with other members of your WG to coordinate decisions about
+   whether issues should move forward
+1. **DO** make decisions on whether or not issues should proceed forward with implementations or RFCs
+1. **DO** assign the [correct labels][issue-process] to issues
```

Comment:
that assumes we will have the privilege to do so. is that something we will have to enable?

Reply:
Naively, we should make WGs Teams within the PowerShell organization and give them Triage permissions within the PS org. However, we may not be able to add external collaborators without adding them to the PS org

Given the fullness of time:
* if we're can add external contributors to the PS org (this may not be something we can do, we'll go and check), we should rework permissions such that WG teams only have access to public repos
* we can also use a bot that enables trusted users to use comment-based commands to label on their behalf

In the intermediate, we can simply add external contributors directly to the PS repo with Triage permissions.

Reply:
I think we need initially auto set `untriaged` label until WG makes a conclusion for the issue.

Reply:
Our thinking thus far has been for WG members to be the ones to add an `Issue - *` label as the "2nd order triage", but I kind of like this better. Will raise it up with the team

Reply:
Related discussion with @daxian-dbw https://github.com/PowerShell/PowerShell/pull/14645#issuecomment-764853472

Reply:
You can give triage without making the part of the org.  I have done it in PowerShell-Docker

Reply:
Mhm. I have triage permissions. Dunno how it's done, but it's doable. Think it has to be done on an individual basis, though. For the purposes of WGs it might make more sense to have people be a member of the org under a specific WG "team"/group with appropriate permissions set. Less hassle, I would think.